pkgs/by-name/mi/mihomo-party/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  wrapGAppsHook3,
+  autoPatchelfHook,
+  nss,
+  nspr,
+  alsa-lib,
+  openssl,
+  webkitgtk,
+  udev,
+  libayatana-appindicator,
+  libGL,
+}:
+let
+  pname = "mihomo-party";
+  version = "1.4.0";
+  src = fetchurl {
+    url = "https://github.com/mihomo-party-org/mihomo-party/releases/download/v${version}/mihomo-party-linux-${version}-amd64.deb";
+    sha256 = "/iC63QXEQ8DnrHTkgYO6Audxib4GqScDIk76kDypN6E=";
```

Could you recalculate this as an SRI hash?

```suggestion
    hash = "sha256-????";
```